### PR TITLE
Run 'safe' 2to3 fixers and convert print to PtDebugPrint

### PR DESCRIPTION
**Pull request by aqua-uru**
Run those 2to3 fixers which should be safe. Additionally, change `print` calls to `PtDebugPrint`.

Included fixers: `print`, `exec`, `import`, `has_key`, `repr`, `funcattrs`, `raise`, `except`.

Excluded fixers and reasons. Links to commits which apply the fix.
- [unicode](https://github.com/aqua-uru/moul-scripts/commit/e319e6339d94a555a67150fe12b9582efdcc29e1): scary string handling changes
- [dict](https://github.com/aqua-uru/moul-scripts/commit/3a46884ef0ebc5cf5f497ccfad4b85516fe05b87), [xrange](https://github.com/aqua-uru/moul-scripts/commit/352ae9910e7e417d50683aae3725b00f6792849c): changes behaviour (list vs generators) so should only happen when switching to py3. Also generators should be introduced in the py2 code.
- [imports](https://github.com/aqua-uru/moul-scripts/commit/6f87c0d83e66206a83a42eda35e2425f4a7c59df): changes `cPickle` to `pickle` which in py2 is a speed difference.
- [long](https://github.com/aqua-uru/moul-scripts/commit/bcdcf174b1329fed43417d73ff2420d59df50f70): scary type handling changes
- [metaclass](https://github.com/aqua-uru/moul-scripts/commit/ac745bd82c22f6a8e208d2fb57813e63d5efb097): creates non-py2 code.

The system/ directory is excluded from changes.

### CODE REVIEW

**Comment by Hoikas** [comment]
It looks like there are now conflicts with this PR.

**Comment by aqua-uru** [comment]
Conflixed!

File: Python/xOptionsMenu.py [hunk 1]
```diff
@@ -1746,18 +1746,18 @@ def setNewChronicleVar(self, chronicleVar, value):
...
+            PtDebugPrint("%s:\tyour current value for %s is %s" % (kModuleName,chronicleVar,entry.chronicleGetValue()))
 
     def getChronicleVar(self, chronicleVar):
         kModuleName = "Personal"
         kChronicleVarType = 0
         vault = ptVault()
         entry = vault.findChronicleEntry(chronicleVar)
-        print "getChronicleVar.chronicleVar: " + chronicleVar
+        PtDebugPrint("getChronicleVar.chronicleVar: " + chronicleVar)
         #print "getChronicleVar.Entry: " , entry
```
Comment (Hoikas) [inline Python/xOptionsMenu.py:1761]: Commented turd

File: Python/xOptionsMenu.py [hunk 2]
```diff
@@ -1024,7 +1024,7 @@ def OnGUINotify(self,id,control,event):
             #~ print "navigation event = ", event
             #~ print "kShowHide = ",kShowHide, " kAction = ",kAction, " kValueChanged = ", kValueChanged
```
Comment (Hoikas) [inline Python/xOptionsMenu.py:1025]: Commented turd

File: Python/xLinkingBookGUIPopup.py [hunk 3]
```diff
@@ -293,7 +293,7 @@ def OnNotify(self,state,id,events):
                             PtSetOfferBookMode(self.key, self.IGetAgeFilename(), self.IGetAgeInstanceName())
                             PtSetShareSpawnPoint(self.IGetAgeSpawnPoint())
                             if self.IGetAgeFilename() == "PelletBahroCave":
-                                print "using Adam's plasma-magical 'PtSetShareAgeInstanceGuid' with pelletcaveguid = ",pelletCaveGUID
+                                PtDebugPrint("using Adam's plasma-magical 'PtSetShareAgeInstanceGuid' with pelletcaveguid = ",pelletCaveGUID)
```
Comment (Hoikas) [inline Python/xLinkingBookGUIPopup.py:296]: Disclaimer: this is not talking about me. I would have never written this shameless evil.

Comment (aqua-uru) [reply Python/xLinkingBookGUIPopup.py:296]: But it's plasma-magical! Must be good.

**Comment by aqua-uru** [comment]
> I think you need to make a pass through and look for any commented out print statements that may be lurking around. I saw a few just glancing at the changes.

2to3 ignores comments, hence they were skipped (it might affect other fixers as well). I've done a search-replace for the commented prints and amended the commit with those. I also reverted the ones in the three files you mentioned.

**Pull request merged**